### PR TITLE
Scala in Android - Preparing the environment

### PR DESCRIPTION
**Pull request by fedefernandez**
Adds a project structure for the post "Scala in Android - Preparing the environment"

Please @javipacheco could you review? Thanks

### CODE REVIEW

File: src/main/scala/com/fortysevendeg/scala/android/SampleActivity.scala [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.fortysevendeg.scala.android
+
+import android.app.Activity
+import android.os.Bundle
+import android.widget.TextView
+
+class SampleActivity extends Activity {
```
Comment (javipacheco) [inline src/main/scala/com/fortysevendeg/scala/android/SampleActivity.scala]: We should use a `TypedResources` 

```
class SampleActivity extends Activity with TypedActivity {
...
   findView(TR.textview).setText(R.string.hello_world)
...
}
```

File: project/plugins.sbt [hunk 2]
```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.hanhuy.sbt" % "android-sdk-plugin" % "1.3.17")
```
Comment (javipacheco) [inline project/plugins.sbt]: version `1.3.18`

**Comment by javipacheco** [comment]
We should add Copyright in all files

**Comment by fedefernandez** [comment]
Comments addressed @javipacheco could you take a look?

File: src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    package="com.fortysevendeg.scala.android"
+    android:versionCode="1"
+    android:versionName="1.0">
+
+    <uses-sdk
+        android:minSdkVersion="14"
+        android:targetSdkVersion="21"/>
+
+    <application
+        android:theme="@android:style/Theme.Holo.Light">
```
Comment (javipacheco) [inline src/main/AndroidManifest.xml]: Why Theme Holo? If you don't use theme attribute, android should set the default theme (Lollipop >= 21, Holo < 21)

It's not an important comment and it's not necessary change it

**Comment by javipacheco** [comment]
LGTM!

File: src/main/AndroidManifest.xml (hunk 3 above)
Comment (fedefernandez) [inline src/main/AndroidManifest.xml]: Sorry my bad, I will change it and do the meege. Thanks

**Pull request merged**